src/components/AudioTransportBar/AudioTransportBar.js

```diff
@@ -202,4 +225,21 @@ class AudioTransportBar extends Component {
   }
 }
 
-export default AudioTransportBar;
+const mapStateProps = state => ({
+  zoom: state.viewState[VIEWSTATE.ZOOM],
+});
+
+const mapDispatchToProps = {
+  zoomIn,
+  zoomOut,
+  resetZoom,
+};
+
+//export default AudioTransportBar;
+
+export default compose(
+  connect(
+    mapStateProps,
+    mapDispatchToProps
+  )
+)(AudioTransportBar);
```

Comment:
If we are going to turn this stateless component into a connected one, I think we should move it to `./src/containers` and add the rest of the connected fields (currently passed down in `VariationsMainView`) or move these fields to there

Reply:
I believe we fixed this in newer commits.